AxonDeepSeg/train_network.py

```diff
@@ -89,9 +89,8 @@ def train_model(
     rotation = config["da-2-random_rotation-activate"]
     elastic = config["da-3-elastic-activate"]
     flipping = config["da-4-flipping-activate"]
-    if "da-5-gaussian_blur-activate" in config:
-        gaussian_blur = config["da-5-gaussian_blur-activate"]
-    elif "da-5-noise_addition-activate" in config:
+    gaussian_blur = False
+    if "da-5-noise_addition-activate" in config:
         # Preserved for retrocompatibility with old configs
         gaussian_blur = config["da-5-noise_addition-activate"]
```

Comment:
If we're removing this functionality for a good reason (which you listed in issue #288), maybe it would be best to   remove the retrocompatibility here, and simply throw a warning that the old config had this setting on, and we don't recommended it so it's set to False. What do you think @mariehbourget ?

Reply:
Yes, I agree with you. I'm not sure if it is even necessary to throw a warning. I haven't found any example of config files with that key True (in the current models of previous releases models). Maybe it was used for development only? Let me know what would be the best practice here and I'll make the changes.